include/http/request_handler.js

```diff
@@ -356,6 +356,20 @@ module.exports = function RequestHandlerModule(pb) {
                 }
 
                 this.resp.setHeader('content-type', contentType);
+
+                const secureCookies = this.req && this.req.siteObj && this.req.siteObj.secureCookies;
+                if (secureCookies) {
+                    const cookies = this.resp.getHeader('set-cookie');
+                    (cookies || []).forEach((cookie, index) => {
```

Comment:
1. So, I understand this is a safety check, but doesn't make sense to forEach an empty array. We can move it outside and if it is an empty array we do not do anything.
2.

Reply:
@CB-JackLi 
```
if (cookies) {
    cookies.forEach
}
```
is the same as `(cookies || [])`, I like the second way because it would reduce a block.